apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -281,6 +290,21 @@ void _wt_print_conninfo(wifi_manager_info_s *info)
 	}
 }
 
+int _wt_get_scanned_list(wifi_manager_scan_info_s *slist, char *ssid,
+						 wifi_manager_ap_auth_type_e *atype,
+						 wifi_manager_ap_crypto_type_e *ctype)
+{
+	int ssid_len = strlen(ssid);
```

Comment:
I think we need to check ssid_len vaue like `if (ssid_len == 0) { return -1} ` to avoid wrong memory access when `ssid is NULL`.

Reply:
it's guaranteed that ssid is not null.